projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -209,7 +209,7 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
         });
         this.panelRef.afterExpand.subscribe(() => {
             this._taskContentService.$shouldCreate.pipe(take(1)).subscribe(() => {
-                this._taskContentService.blockFields(!this.canFinish());
+                this._taskContentService.blockFields(!this.canFinish(false));
```

Comment:
Better to remove `boolean` param. Maybe for readability create new function that will return value equal to calling 
`!this.canFinish(false)` right now.